install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,124 +11,248 @@
-link:../../architecture/additional_concepts/storage.html[persistent storage] using
-https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS].
-Some familiarity with Kubernetes and NFS is assumed.
-
-The Kubernetes
-link:../../architecture/additional_concepts/storage.html[persistent volume]
-framework allows administrators to provision a cluster with persistent storage
-and gives users a way to request those resources without having any knowledge of
-the underlying infrastructure. Persistent volumes are not bound to a single
-project or namespace; they can be shared across the OpenShift cluster.
-link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
-volume claims], however, are specific to a project or namespace and can be
-requested by users.
-
-For a detailed example, see the guide for
-https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and
-MySQL using NFS].
-
-[IMPORTANT]
-====
-High-availability of storage in the infrastructure is left to the underlying
-storage provider.
-====
-
-[[nfs-provisioning]]
+
+OpenShift clusters can be provisioned with
+link:../../architecture/additional_concepts/storage.html[persistent storage] using NFS
+Some familiarity with Openshift,
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent volume claims (PVCs)],
+and https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS] is beneficial.
+
+The Openshift persistent volume (PV) framework allows administrators to provision a cluster
+with persistent storage and gives developers the ability to request those resources without
+having specific knowledge of the underlying infrastructure. Persistent volumes are not bound
+to a single project; they can be shared across the entire OpenShift cluster. Persistent volume
+claims (PVCs), however, are specific to a project and are created and used by developers.
+
+(*Note:* "project" and "namespace" are used interchangeably throughout this document)
 
 == Provisioning
-Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision NFS volumes in OpenShift, all that is
-required is a distinct list of NFS servers and paths and the
-`*PersistentVolume*` API.
 
-You must define your persistent volume in an object definition before creating
-it in OpenShift:
+Storage must exist in the underlying infrastructure before it can be mounted as a volume in
+OpenShift. To provision NFS volumes a list of NFS servers and export paths are all that is required.
 
-.Persistent Volume Object Definition Using NFS
-====
+*Note:*
+Each NFS volume must be mountable by all schedulable nodes in the cluster.
 
-[source,yaml]
-----
-apiVersion: "v1"
-kind: "PersistentVolume"
+A PV spec file is usually defined before creating the PV:
+```
+apiVersion: v1
```

Comment:
Example files should use formatting as seen in https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#examples

Basically:

```
====
[source,yaml]
----
Stuff
in
a file.
----
====